### ISSUE TITLE
Cannot read property 'insert' of undefined

### ISSUE THREAD

**Issue by Nino3245** [LLK/scratch-gui#9337]
### Expected Behavior

NPM should install everything requiered.

### Actual Behavior

It fails with "Cannot read property 'insert' of undefined" after installing "semantic-release@19.0.5"

### Steps to Reproduce

1. Git Clone the repository to a folder,
2. Cd into it,
3. And run, "npm install"

### System Details
Ubuntu 22.04 (Virtual Machine) Linux Terminal (Ran as Root)

**Screenshots**
![ezgif com-gif-maker (3)](https://github.com/scratchfoundation/scratch-gui/assets/94793668/28236adb-1e04-4359-b6fe-84e1e0ec0968)

**Comment by ajskateboarder** [comment]
I don't believe the dependencies will install on Node v12. Try updating Node to v16 and running `npm install` again
